### DISCUSSION TITLE
If bulk_create doesn't support IntField Primary Keys, how or why would I even use it?

### DISCUSSION THREAD

**Discussion by PancakePuncher** [tortoise/tortoise-orm#1579] [Q&A]
I am curious if anyone has a solution to this?

Other then a for loop on a single "create" method.

The issue is that when I attempt to do a bulk_create on the table Schema below. I get an error stating that I am missing the "_custom_primary_key". I do not expect the incoming objects to include the ID of the record. I want this to be generated every time a record is created.

My Objects come in like this using Pydantic Models:
```python
[
Question(question_text="", question_category="", question_type="", question_scope=""), 
Question(question_text="", question_category="", question_type="", question_scope="")
]
```

I attempted filling in the "pk_question_id" with a None to see if that would force it to generate it, but no luck.

Schema For Reference
```python
class Questions(Model):
    class Meta:
        table = "questions"

    pk_question_id = fields.IntField(pk=True)
    question_text = fields.TextField(null=False)
    question_category = fields.CharField(max_length=25, null=False)
    question_type = fields.CharField(max_length=25, null=False, default="normal")
    question_creation_date = fields.DatetimeField(auto_now_add=True, null=False)
    question_last_edit = fields.DatetimeField(
        auto_now=False, auto_now_add=True, null=False
    )
    question_scope = fields.CharField(max_length=25, null=False)
```

But the bulk_create documentation says this...

![image](https://github.com/tortoise/tortoise-orm/assets/3018272/0bd438d8-c619-4e97-9dac-0b1c47e4a72c)

This seems kind of insane to me -- is their a way to work around this behavior? Because so far I haven't found any documentation on this.

**Comment by vlakius** [comment]
It seems really strange to me.
The documentation probably refers to the fact that if you try to set the primary key manually, it will not be entered in the query.
I deduce this from the fact that the primary key cannot be null if it is required in the table.
Furthermore, I have used bulk_create several times without any problems, but I have never tried to do it by manually entering the primary key.

what happen if you run this ? 
 ```python
questions = [
Questions(question_text="", question_category="", question_type="", question_scope=""), 
Questions(question_text="", question_category="", question_type="", question_scope="")
]

await Questions.bulk_create(questions)
```
